README.md

```diff
@@ -240,6 +240,69 @@ const myUiManager = new UIManager(player, myUi, myUiConfig);
 
 All the configuration properties are optional. If `metadata` is set, it overwrites the metadata of the player configuration. If `recommendations` is set, a list of recommendations is shown in the `RecommendationOverlay` at the end of playback. For this to work, the UI must contain a `RecommendationOverlay`, like the default player UI does.
 
+### UI Localization
+UI can be localized by passing `localization` object to *uiConfig* while initializing UIManager. By default the player UI supports `english` and `german` and will set the language to `browsers preferred language` by default (if language has a vocabulary defined) if this behavior is not intended, you can easily disable it by adding `disableBrowserLanguageDetection: true` parameter to your Localization Config...
+
+```ts
+interface LocalizationConfig {
+  language?: 'en' | 'de' | string;
+  fallbackLanguages?: string[];
+  disableBrowserLanguageDetection?: boolean;
+  translations?: BitmovinPlayerUiTranslations;
```

Comment:
Apart from the previous open question here, do we even need the `translations` property? Why don't we make 'language: string | Vocabulary' - if it is a `string` we try to load a JSON file with this name, if it is a `Vocabulary` object then there's nothing to do.

Reply:
the `translations` there so that user can add multiple languages. 
yes we set only one vocabulary from the `language` user sets but we also check `browserPrefferedLang` so if the user has already added the language we can set it. so imo that doesn't makes sense to have `string | Vocabulary` which than would make localization very limited...

Reply:
for the prev. comment. we only use `Localization` and `translations` 
which makes sense because the config is made for `Localization` and the `translations` object is contains the `translations` for the languages. other option would be to renam it to `languageKeyVocabularyPair`

Reply:
Localization is usually concerned with more aspects that just pure translation that we do here, e.g. number/time formatting, right-to-left. But sure we should not limit our possibilities with a name too narrow.

Reply:
> for the prev. comment. we only use `Localization` and `translations`

`i18n`

Reply:
i18n is a standard naming in this domain (by standard I mean used by most of the people who does localization), which represents the process and planning of i(nternationalizatio)n process which includes Localization & Translation of words, sentences ..etc.. If we want to look at the semantics it makes sense to call the process(class) `i18n` and to include `localization` and `translation`s inside that class.
As I checked the libraries this was the namings they used and that was the reason I choose to name it i18n.It can also be renamed to `Localization` if you think it might cause confusion

Reply:
but in this case I find it more weird to call `localization.localize` or `localization.getLocalizableCallback` ?